### PR TITLE
plugins: add willitconnect

### PR DESCRIPTION
**Pull request by gambtho**
Convenience plugin to provide access to https://github.com/krujos/willitconnect

Current version supports using proxies as well as setting a non-default route for the willitconnect application

### CODE REVIEW

File: repo-index.yml [hunk 1]
```diff
@@ -512,7 +512,7 @@ plugins:
   version: 1.0.0
   created: 2015-11-11
   updated: 2015-11-11
-  company: 
+  company:
```
Comment (dkoper) [inline repo-index.yml]: are you mixing your feature with cleanup of other code?

Comment (gambtho) [inline repo-index.yml]: that wasn't an intentional change -- i would guess atom automatically removed that space

Comment (gambtho) [inline repo-index.yml]: Resubmitted the PR with that change removed -- just my plugin addition now.

CLA submitted as well

Comment (dkoper) [inline repo-index.yml]: Clean up and squash commits please. Also, please make whoever you submitted the CLA to aware of this PR and add a comment that they've received it? Cheers.

Comment (gambtho) [inline repo-index.yml]: Cleaned, squashed and requested, thanks for the quick response.

**Comment by chipchilders** [comment]
CLA received @dkoper

**Comment by chendrix** [comment]
@gambtho It looks like for some reason the link for the windows release is broken, looks like some kind of github/s3 issue? 

If you also go to [your github release](https://github.com/gambtho/cf_will_it_connect_plugin/releases/tag/v1.1.0) and click on the windows upload, it shows the same issue.

How would you like the proceed? Everything else checked out fine, and besides this issue your plugin is ready to be added.

**Comment by gambtho** [comment]
Sorry about that, re-uploaded the windows binary with a shorter name, seems to be all good now.

**Pull request merged**